app-prod.yaml

```diff
@@ -33,10 +33,10 @@ spec:
     max: 4
   resources:
     limits:
-      memory: 6096Mi
```

Comment:
Jeg har hørt en gang at limit og request bør være lik for minne og [denne tilfeldige bloggposten](https://loft.sh/blog/how-to-set-up-kubernetes-requests-and-limits/#:~:text=Always%20Set%20Equal%20Memory%20Requests%20and%20Limits&text=You%20will%20usually%20want%20to,memory%20allocation%20systems%20outside%20Kubernetes) mener det samme.

Reply:
topp, mener å huske at vi økte for å få igjennom grensesnittavstemming, men den problematikker har vi vel delvis fikset nå så sikkert ok å justere ned. Er vel slik at hvis pod bruker over limits så knerter kubernetes pod'en